runner/environment.go

```diff
@@ -0,0 +1,27 @@
+package runner
+
+import (
+	"bufio"
+	"os"
+	"os/exec"
+)
+
+func (e *Engine) modifyEnvironment(c *exec.Cmd) {
```

Comment:
Side remark, but is the "bare return" you apply in this method a common thing in this repository

revive discourages using it, and I agree with them

https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#bare-return

Reply:
It might be a better idea to return an error, you're right.

Reply:
By bare return, I meant using `return c`everywhere and not `return`

Reply:
But your new implementation is better, I like it 👍